server.py

```diff
@@ -218,6 +254,28 @@ def after_this_request(f):
 #  Helper method section
 ###############################################
 
+# Update metadata on this server using info from another server
+#
+# params:
+#   server: server to query/ask
+#   uuid: uuid to query for
+def update_metadata_from_another_server(server, uuid):
+    url = 'http://%s/metadata?%s' % (server, urllib.urlencode({ 'uuid': uuid }))
+    r = requests.get(url)
+    response = json.loads(r.text)
+    metadata.update_file_stored(response['uuid'], response['server'], response['file_size'])
+    return response
+
+# Get the file size of a file
+#
+# params:
+#   file_path: absolute path to file
+def get_file_size(file_path):
```

Comment:
Hmm. in line 57 of this file, it's used in `write_file`, before it's ever even written to the metadata, so I think we need to read the file.